frontend_server/package_incremental.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+# Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+import argparse
+import os
+import sys
+import shutil
+
+# The list of packages copied from the Dart SDK.
+PACKAGES = [
+    "vm", "build_integration", "kernel", "front_end", "frontend_server",
+]
+
+VM_PUBSPEC = r'''name: vm
+version: 0.0.1
+environment:
+  sdk: ">=2.2.2 <3.0.0"
+
+dependencies:
+  front_end: any
+  kernel: any
+  meta: any
+  build_integration: any
+'''
+
+BUILD_INTEGRATION_PUBSPEC = r'''name: build_integration
+version: 0.0.1
+environment:
+  sdk: ">=2.2.2 <3.0.0"
+
+dependencies:
+  front_end: any
+  meta: any
+'''
+
+FRONTEND_SERVER_PUBSPEC = r'''name: frontend_server
+version: 0.0.1
+environment:
+  sdk: ">=2.2.2 <3.0.0"
+
+dependencies:
+  args: any
+  path: any
+  vm: any
+'''
+
+KERNEL_PUBSPEC = r'''name: kernel
+version: 0.0.1
+environment:
+  sdk: '>=2.2.2 <3.0.0'
+
+dependencies:
+  args: any
+  meta: any
+'''
+
+FRONT_END_PUBSPEC = r'''name: front_end
+version: 0.0.1
+environment:
+  sdk: '>=2.2.2 <3.0.0'
+dependencies:
+  kernel: any
+  package_config: any
+  meta: any
+'''
+
+PUBSPECS = {
```

Comment:
Does the engine repo use the google3 style (4 space indent) or the chrome style (2 space indent) for python? @chinmaygarde

Reply:
I switched to 2-space, that looks like the rest of the repo